File: engine/model/src/main/scala/net/server/channel/packet/command/admin/KillMonsterPacket.scala
package net.server.channel.packet.command.admin

import net.server.MaplePacket

class KillMonsterPacket( private var _mode: Byte,  private var _mobToKill: Int,  private var _amount: Int) extends BaseAdminCommandPacket(_mode = _mode) {
     def mobToKill: Int = _mobToKill
     def amount: Int = _amount
}

File: engine/engine-model/src/main/java/net/server/channel/packet/command/admin/KillMonsterPacket.java
package net.server.channel.packet.command.admin;

public class KillMonsterPacket extends BaseAdminCommandPacket {
   private final Integer mobToKill;

   private final Integer amount;

   public KillMonsterPacket(Byte mode, Integer mobToKill, Integer amount) {
      super(mode);
      this.mobToKill = mobToKill;
      this.amount = amount;
   }

   public Integer mobToKill() {
      return mobToKill;
   }

   public Integer amount() {
      return amount;
   }
}
